mainmatter.md

```diff
@@ -147,17 +126,17 @@ Figure 3: Example JWT for `DPoP-Binding` header.
 If the authorization server receives a `DPoP-Binding` header in a
 token request, the authorization server MUST check that
 
- * the header value is a well-formed JWT,
- * all required claims are contained in the JWT,
- * the algorithm in the header of the JWT is supported by the
+ 1. the header value is a well-formed JWT,
+ 2. all required claims are contained in the JWT,
```

Comment:
The WebAuthn MarkDown sources use the convention that all elements of numbered lists use "1.".  Markdown will automatically then number the lists, which makes insertions and deletions of items easier.  Consider doing that.